tests/calculators/test_orca.py

```diff
@@ -131,15 +132,15 @@ def test_orca_geometry_optimisation(tmp_path):
     inputs = ConfigSet(atoms)
     outputs = OutputSpec()
 
-    calc = ORCA(directory=home_dir, 
+    calc = ORCA(workdir=home_dir, 
                 keep_files = "default", 
                 mult=1, 
                 task="opt")
 
 
-    generic.run(inputs=inputs, outputs=outputs, calculator=calc, properties=["energy", "forces"], output_prefix="orca_", num_python_subprocesses=0)
+    generic.run(inputs=inputs, outputs=outputs, calculator=calc, properties=["energy", "forces"], output_prefix="orca_", autopara_info=AutoparaInfo(num_python_subprocesses=0))
```

Comment:
Why is this intentionally turning off pool autoparallelization? Is it not working with orca?